### PR TITLE
HW3 task 2

### PR DESCRIPTION
**Pull request by ValeriL**

### CODE REVIEW

File: Homework/homework3/test_task2.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from homework3.task2 import speed_up_calculate
+
+
+def test_speed_up_calculate():
```
Comment (Cjkjvfnby) [inline Homework/homework3/test_task2.py:4]: Please test calculation result, not time.  
To test time you need to have context.  So this test may fail on some machines, or even worse fail from time to time.

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]

File: Homework/homework3/test_task2.py (hunk 1 above)
Comment (ValeriL) [reply Homework/homework3/test_task2.py:4]: Okay, understood.
But do I need somehow show that the calculations take less than a minute how it should be owing to the task?

Comment (Cjkjvfnby) [reply Homework/homework3/test_task2.py:4]: It's ok for me that you measure it manualy.  Time measure and unittesting are different type of testing and require different approaches.

Comment (Cjkjvfnby) [reply Homework/homework3/test_task2.py:4]: Keep it fast and simple.
`assert speed_up_calculate(500) ==  expected`

**Review by Cjkjvfnby — APPROVED** [review]

**Pull request merged**